source/ClientRequest/AMDSClientRelativeCountPlusCountDataRequest.cpp

```diff
@@ -0,0 +1,76 @@
+#include "source/ClientRequest/AMDSClientRelativeCountPlusCountDataRequest.h"
+
+#include "source/AMDSDataStream.h"
+
+AMDSClientRelativeCountPlusCountDataRequest::AMDSClientRelativeCountPlusCountDataRequest(QObject *parent) :
+	AMDSClientDataRequest(parent)
+{
+	setRequestType(AMDSClientRequestDefinitions::RelativeCountPlusCount);
+
+	setRelativeCount(0);
+	setCount(1);
+}
+
+AMDSClientRelativeCountPlusCountDataRequest::AMDSClientRelativeCountPlusCountDataRequest(ResponseType responseType, const QString &socketKey, const QString &bufferName, bool includeStatusData, const quint64 relativeCount, quint64 count, const AMDSBufferGroupInfo &bufferGroupInfo, QObject *parent):
+	AMDSClientDataRequest(socketKey, QString(), AMDSClientRequestDefinitions::RelativeCountPlusCount, responseType, bufferName, includeStatusData, bufferGroupInfo, parent)
+{
+	setRelativeCount(relativeCount);
+	setCount(count);
+}
+
+AMDSClientRelativeCountPlusCountDataRequest::~AMDSClientRelativeCountPlusCountDataRequest()
+{
+
+}
+
+AMDSClientRelativeCountPlusCountDataRequest::AMDSClientRelativeCountPlusCountDataRequest(const AMDSClientRelativeCountPlusCountDataRequest &other):
+	AMDSClientDataRequest(other)
+{
+	(*this) = other;
+}
+
+AMDSClientRelativeCountPlusCountDataRequest& AMDSClientRelativeCountPlusCountDataRequest::operator =(const AMDSClientRelativeCountPlusCountDataRequest &other)
+{
+	if(this != &other){
+		AMDSClientDataRequest::operator =(other);
+		setRelativeCount(other.relativeCount());
+		setCount(other.count());
+	}
+	return (*this);
+}
+
+bool AMDSClientRelativeCountPlusCountDataRequest::writeToDataStream(AMDSDataStream *dataStream) const
+{
+	if(!AMDSClientDataRequest::writeToDataStream(dataStream))
```

Comment:
Would it make more sense to have status enums for these rather than bools?  That way you can introspect the failure of the data server more specifically.